packages/core/README.md

```diff
@@ -149,6 +150,21 @@ await Config.register({
 });
 ```
 
+#### Types conversion
+
+<a name="loading_adapters_conversion"></a>
+
+When loading configuration from predefined objects it's handy to
+to disable the default behavior of implicit properties types conversion.
+
+```ts
+await Config.register({
+  template: AppSettings,
+  autoConvertTypes: false,
```

Comment:
misc: using the same flag name as in underlying library could have nice benefits. I could even assume it's passed right to underlying library. Maybe it's worth changing this field name to `enableImplicitConversion`? Also, it gives a bit more of a context, since there are implicit and explicit type conversions out there.

Reply:
Yeah. My intention was to decouple lib api from `class-transformer` api. But it's a leaking abstraction anyway.